src/box/lua/net_box.c

```diff
@@ -1151,6 +1151,28 @@ netbox_transport_connect(struct netbox_transport *transport)
 	return -1;
 }
 
+/*
```

Comment:
`netbox_transport_connect` is also executed inside worker fiber: `coio_connect_timeout` yields in `coio_connect_addr`, after that it tries to read with `coio_readn_timeout` (which also yields, BTW). It looks like we should use `netbox_worker_check_after_yield` in `netbox_transport_connect` too

Reply:
Fixed.